README.md

```diff
@@ -163,57 +180,28 @@ and select the profiles you want to build.
 
 ### Building Eclipse Kura™ Containers
 
-The kura™ container build process currently only supports x86 containers. Following the instructions below will build two containers. One based on Alpine Linux ```kura-alpine-x86_64```, and another on Ubi8 ```kura-ubi8-x86_64```.
+The kura™ container build process currently only supports x86 containers. Following the instructions below will build two containers. One based on Alpine Linux `kura-alpine-x86_64`, and another on Ubi8 `kura-ubi8-x86_64`.
```

Comment:
Eclipse Kura

Reply:
Addressed in: 3fafedaed27d31d091bd64e67c809a32d5e77ac3